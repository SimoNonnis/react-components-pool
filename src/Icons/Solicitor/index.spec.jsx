/* eslint-env jest */
import React from 'react';

import Solicitor from './';

describe('Test <Solicitor /> icon', () => {
  it('should render Solicitor icon correctly with default props', () => {
    const wrapper = shallow(<Solicitor />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render Solicitor icon correctly with props', () => {
    const wrapper = shallow(
      <Solicitor fill="#a4a1a1" className="testClassname" height={30} width={30} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
