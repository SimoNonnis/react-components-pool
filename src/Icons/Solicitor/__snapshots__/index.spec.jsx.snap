// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Solicitor /> icon should render Solicitor icon correctly with default props 1`] = `
<svg
  className="solicitorIcon"
  height={40}
  viewBox="0 0 30 30"
  width={40}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Solicitor Icon
  </title>
  <path
    className="solicitorIconPath"
    d="M27.73,14.5,25,7.15A1,1,0,0,0,24.87,7H25a1,1,0,0,0,0-2H5A1,1,0,0,0,5,7h.17a1,1,0,0,0-.08.15L2.31,14.5H1v1a5,5,0,0,0,10,0v-1H9.73L7,7.15A1,1,0,0,0,6.87,7H14V24H11.35a1.25,1.25,0,0,0,0,2.5h7.5a1.25,1.25,0,1,0,0-2.5H16V7h7.17a1,1,0,0,0-.08.15L20.31,14.5H19v1a5,5,0,0,0,10,0v-1ZM6,18.5a3,3,0,0,1-2.83-2H8.83A3,3,0,0,1,6,18.5Zm1.59-4H4.45L6,10.33ZM24,10.33l1.57,4.17H22.45Zm0,8.17a3,3,0,0,1-2.83-2h5.66A3,3,0,0,1,24,18.5Z"
    fill="currentColor"
  />
</svg>
`;

exports[`Test <Solicitor /> icon should render Solicitor icon correctly with props 1`] = `
<svg
  className="testClassname"
  height={30}
  viewBox="0 0 30 30"
  width={30}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Solicitor Icon
  </title>
  <path
    className="solicitorIconPath"
    d="M27.73,14.5,25,7.15A1,1,0,0,0,24.87,7H25a1,1,0,0,0,0-2H5A1,1,0,0,0,5,7h.17a1,1,0,0,0-.08.15L2.31,14.5H1v1a5,5,0,0,0,10,0v-1H9.73L7,7.15A1,1,0,0,0,6.87,7H14V24H11.35a1.25,1.25,0,0,0,0,2.5h7.5a1.25,1.25,0,1,0,0-2.5H16V7h7.17a1,1,0,0,0-.08.15L20.31,14.5H19v1a5,5,0,0,0,10,0v-1ZM6,18.5a3,3,0,0,1-2.83-2H8.83A3,3,0,0,1,6,18.5Zm1.59-4H4.45L6,10.33ZM24,10.33l1.57,4.17H22.45Zm0,8.17a3,3,0,0,1-2.83-2h5.66A3,3,0,0,1,24,18.5Z"
    fill="#a4a1a1"
  />
</svg>
`;
